js/util.js

```diff
@@ -8,10 +8,16 @@ const generateRandomNumber = (min, max) => {
   return min >= 0 ? min : 0;
 };
 
-export const getRandomInteger = (min, max) => Math.floor(generateRandomNumber(min, max));
+const addClassHidden = ( element, className ) => {
```

Comment:
```suggestion
const addClassHidden = (element, className) => {
```

все привыкли писать вот так) ну и везде лучше писать в одном стиле)